tests/integration_tests/test_applications_from_config.py

```diff
@@ -134,6 +135,42 @@ def compare_ecsv_files(file1, file2, tolerance=1.0e-5):
             assert np.allclose(table1[col_name], table2[col_name], rtol=tolerance)
 
 
+def assert_file_type(file_type, file_name):
```

Comment:
Should we include ecsv file here as well?

Reply:
Yes, and probably for any other file types we are using. I think this is beyond this pull request (it is already a 'bonus' on this one).